bash_completions.go

```diff
@@ -617,6 +617,7 @@ func writeCmdAliases(buf io.StringWriter, cmd *Command) {
 	WriteStringAndCheck(buf, `    fi`)
 	WriteStringAndCheck(buf, "\n")
 }
+
```

Comment:
It seems that some of these changes to `.go` sources are motivated by changing the golangci-lint configuration, which is unrelated to the topic of this PR. Please, keep these changes in a separated commit and/or PR.